curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-basic-dom-manipulation-by-building-a-role-playing-game/65f664b393ed8547a20bafd8.md

```diff
@@ -0,0 +1,259 @@
+---
+id: 65f664b393ed8547a20bafd8
+title: Step 119
+challengeType: 0
+dashedName: step-119
+---
+
+# --description--
+
+Now that you have a better understanding on how to access items from the monsters array, you can delete your console statement and the `fighting=2` line.
```

Comment:
It would be better to open a separate issue to decide on if the word console should be wrapped in backticks or if console statement should be wrapped in backticks. Or if no backticks are necessary.

Then once the JS team reaches a consensus, it can be opened up for contribution and will be cleaner to audit the project and review those changes, then in this PR.